horovod/spark/lightning/estimator.py

```diff
@@ -162,6 +162,10 @@ class TorchEstimator(HorovodEstimator, TorchEstimatorParamsWritable,
                                'Cache the data in memory for training and validation.',
                                typeConverter=TypeConverters.toBoolean)
 
+    num_gpus = Param(Params._dummy(), 'num_gpus',
+                     'number of gpus per process.')
```

Comment:
Can you explain and put a short description why we need to tell num_gpus per process for lightning?
I don't know how multi-gpus per single process/rank works for horovod.

Reply:
It is a parameter required by lightning trainer.